tests/src/test/java/ApplicationTests.java

```diff
@@ -290,6 +290,120 @@ void liquidJunctionOutput(){
         assertTrue(tank.build.liquids.current() == Liquids.water, "Tank has no water");
     }
 
+    @Test
+    void liquidRouterOutputAll() {
+        world.loadMap(testMap);
+        state.set(State.playing);
+        Tile source = world.rawTile(4,0), router = world.rawTile(4, 2), conduitUp1 = world.rawTile(4,1),
+                conduitLeft = world.rawTile(3,2), conduitUp2 = world.rawTile(4, 3), conduitRight = world.rawTile(5, 2),
+                leftTank = world.rawTile(1, 2), topTank = world.rawTile(4,5), rightTank = world.rawTile(7, 2);
+
+        source.setBlock(Blocks.liquidSource, Team.sharded);
+        source.build.configureAny(Liquids.water);
+        conduitUp1.setBlock(Blocks.conduit, Team.sharded, 1);
+        router.setBlock(Blocks.liquidRouter, Team.sharded);
+        conduitLeft.setBlock(Blocks.conduit, Team.sharded,2);
+        conduitUp2.setBlock(Blocks.conduit, Team.sharded, 1);
+        conduitRight.setBlock(Blocks.conduit, Team.sharded, 0);
+        leftTank.setBlock(Blocks.liquidTank, Team.sharded);
+        topTank.setBlock(Blocks.liquidTank, Team.sharded);
+        rightTank.setBlock(Blocks.liquidTank, Team.sharded);
+
+        updateBlocks(200);
+        assertEquals(true, rightTank.build.liquids.currentAmount() > 0);
```

Comment:
Why not simply `assertTrue`?

Reply:
Anuken requested to use assert equals for comparisons

Reply:
assertTrue is more refined in this case though, I'm going to make an adjustment here.

Reply:
Let me know what you think of the adjustment.

Reply:
I'd just wait for Anuken to see. The change is too minor to think much about.